lib/narn-lib.js

```diff
@@ -2,6 +2,14 @@ const fs = require("fs");
 const path = require("path");
 const minimist = require("minimist");
 
+// like yarn add -D a ==> yarn add a -D, yarn add --dev a ==> yarn add a --dev
+let reverseOption = function reverseOption(narnArgs) {
```

Comment:
This function isn't being used anymore - please remove

Reply:
careless as I.